kitchen/modules/weights/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Don't we use headers here?

Reply:
I'd like to know more about headers. Do you have a link to read?

I do know if I remove this line, the build fails.

Reply:
I mean license headers. Probably @shawntabrizi should decide.